openpype/hosts/hiero/plugins/publish/precollect_workfile.py

```diff
@@ -17,9 +18,10 @@ class PrecollectWorkfile(pyblish.api.ContextPlugin):
     order = pyblish.api.CollectorOrder - 0.491
 
     def process(self, context):
+        asset_name = context.data["asset"]
+        if AYON_SERVER_ENABLED:
+            asset_name = asset_name.split("/")[-1]
```

Comment:
Isn't in AYON that we want it with the slashes?

Reply:
Here the variable should represent name, not path.